epub33/core/index.html

```diff
@@ -835,10 +835,10 @@ <h5>Supported Media Types</h5>
 										codecs without fallbacks, although none are technically Core Media Type
 										Resources. Although the Working Group recommends that Reading Systems support at
 										least one of the H.264 [[?H264]] and VP8 [[?RFC6386]] video codecs, support is
-										not a conformance requirement &#8212; a Reading System might support other video
-										codecs, or none at all. EPUB Creators must consider factors such as breadth of
-										adoption, video playback quality, and technology usage royalty requirements when
-										making a choice to include video in either format, or both. </td>
+										not a conformance requirement &#8212; a Reading System might support any video
+										codec, or none at all. EPUB Creators must consider factors such as breadth of
```

Comment:
It might be clearer here just to say "Reading Systems support for video codecs is optional."

But this overlaps with #2101 so I'll merge this PR separately. This is now under the exempt resources section.

Reply:
I'm going to merge this as there's no point waiting on the other PR for the RS spec change. I'll tweak the other PR for the authoring note.